src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -38,11 +44,13 @@ public static bool TryGetMemoryLimit(out ulong limit)
             return false;
         }
 
+        private static ReadOnlySpan<byte> MaxBytes => new byte[] { (byte)'m', (byte)'a', (byte)'x' };
```

Comment:
If the below if block I commented on ends up being the only use of this, then this can be deleted, too.

Reply:
Removed.